packages/grafana-ui/src/components/Graph/Graph.tsx

```diff
@@ -157,10 +162,12 @@ export class Graph extends PureComponent<GraphProps> {
   }
 
   render() {
-    const { height } = this.props;
+    const { height, series } = this.props;
+    const noDataToBeDisplayed = series.length === 0;
     return (
       <div className="graph-panel">
         <div className="graph-panel__chart" ref={e => (this.element = e)} style={{ height }} />
+        {noDataToBeDisplayed && <div className="datapoints-warning flot-temp-elem">No data points</div>}
```

Comment:
the flot-temp-elem class should not be needed (only for elements inside the flot element)

Reply:
fixed 🙂